draft-ietf-rats-architecture.md

```diff
@@ -127,6 +127,79 @@ Appraisal Policy to make application-specific decisions such as authorization de
 The Attestation Result Appraisal Policy might, for example, be configured in the Relying Party 
 by an administrator.
 
+## Composite Attester {#compositeattester}
+
+A Composite Attester is an entity composed of multiple sub-entities that its
+trustworthiness has to be determined by evaluating all these sub-entities.
+Each sub-entity has at least one Attesting Environment collecting the claims from
+at least one Target Environment and generates its Evidence against its trustworthiness,
+therefore it can be called an Attester. Among these Attesters, there may be only some,
+which can be called Main Attesters, that have the communication functionality
+with the Verifier, and other Attesters are evaluated via their help.
+
+For example, the carrier-grade router is a composite device consisted of multiple slots.
+The trustworthiness of the router depends on all its slots' trustworthiness.
+Each slot has the attesting environment like the TPM or TEE collecting the claims
+of its boot process and generates the Evidence to prove its trustworthiness.
```

Comment:
I think "the" is a superfluous word. "generates Evidence about its trustworthiness"